src/main/io/bf_osd_symbols.h

```diff
@@ -140,8 +138,6 @@
 // Time
 #define BF_SYM_ON_M                    0x9B
 #define BF_SYM_FLY_M                   0x9C
-#define BF_SYM_ON_H                    0x70
```

Comment:
These constants are not in betaflight's font. 0x70 is BF_SYM_SPEED and 0x71 is BF_SYM_TOTAL_DISTANCE

Reply:
One more instance of not seeing the point of being "chained" to BF's proposal. The fonts are there on DJI's table for us to use it. Let's use it!